packages/appium/lib/config.js

```diff
@@ -44,11 +44,23 @@ async function updateBuildInfo (useGithubApiFallback = false) {
   }
 }
 
+/**
+ * Finds the Git metadata dir (see `GIT_META_ROOT`)
+ *
+ * This is needed because Appium cannot assume `package.json` and `.git` are in the same
+ * directory.  Monorepos, see?
+ * @returns {string|void} Path to dir or `undefined` if not found
```

Comment:
we usually put it as `?string`

Reply:
we should lint the docstrings 😄

Reply:
`?string` means "`string` or `null`", and this function never returns `null`.

Reply:
this function might return undefined though

Reply:
isn't that what `void` means?